cumulus_library/study_parser.py

```diff
@@ -315,7 +314,7 @@ def _execute_drop_queries(
             helper.query_console_output(verbose, drop_view_table, progress, task)
 
     def _load_and_execute_builder(
-        self, filename, cursor, schema, verbose, drop_table=False
+        self, filename, cursor, schema, verbose, drop_table=False, parser=None
```

Comment:
Could add type hinting for the parser field too, since you're already importing database classes.